<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="confidence"	default="0.5"/>
    <arg name="image_topic_A"	default="/camera_A/color/image_raw"/>
    <arg name="image_topic_B"	default="/camera_B/color/image_raw"/>
    <arg name="image_topic_C"	default="/camera_C/color/image_raw"/>

    <arg name="caffe_model"	default="$(find py_faster_rcnn_ros)/models/caffe/box_0_1_ZF_faster_rcnn_final.caffemodel"/>

    <!-- <include if="$(arg camera)" file="$(find openni2_launch)/launch/openni2.launch"/> -->
    <node name="detector_manager_node" pkg="py_faster_rcnn_ros" type="detector_manager_multi.py" output="screen" respawn="true">
        <param name="confidence" value="$(arg confidence)"/>
        <param name="caffe_model" value="$(arg caffe_model)"/> 
        <param name="image_topic_A" value="$(arg image_topic_A)"/>
        <param name="image_topic_B" value="$(arg image_topic_B)"/>
        <param name="image_topic_C" value="$(arg image_topic_C)"/>
    </node>

    <!-- <node if="$(arg rviz)" name="rviz" pkg="rviz" type="rviz" args="-d $(find py_faster_rcnn_ros)/rviz/UPennObjectDetector.rviz" /> -->
</launch>
